app/src/test/java/io/intrepid/contest/screens/contestoverview/ContestOverviewPresenterTest.java

```diff
@@ -58,6 +58,36 @@ public void onViewCreatedShouldShowContestNameWhenSuccessfullyRetrievedContestDe
     }
 
     @Test
+    public void onViewCreatedShouldShowRatingGuide() throws Exception {
```

Comment:
Update the test names (this one is missing `When...`)
Choose between `WhenSuccessfullyRetrievedContestDetails` or `WhenSuccessfullyRetrievedContest` and also update the other two tests